keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlUnhappyVetoRepository.kt

```diff
@@ -75,7 +92,7 @@ class SqlUnhappyVetoRepository(
     return sqlRetry.withRetry(READ) {
       jooq.select(UNHAPPY_VETO.RESOURCE_ID)
         .from(UNHAPPY_VETO)
-        .where(UNHAPPY_VETO.RECHECK_TIME.greaterOrEqual(now))
+        .where(UNHAPPY_VETO.RECHECK_TIME.isNull.or(UNHAPPY_VETO.RECHECK_TIME.greaterOrEqual(now)))
```

Comment:
Can’t see the context around this line on the phone, but this needs to be consistent with my comment on one of the checks for skip or recheck being inclusive.